src/index.js

```diff
@@ -6,14 +6,15 @@ import momentLocalizer from './localizers/moment';
 import globalizeLocalizer from './localizers/globalize';
 import viewLabel from './utils/viewLabel';
 import move from './utils/move';
-import { views } from './utils/constants';
+import { views, navigate } from './utils/constants';
 
 Object.assign(Calendar, {
   setLocalizer,
   globalizeLocalizer,
   momentLocalizer,
   label: viewLabel,
   views,
+  navigate,
```

Comment:
I agree. Do you prefer having one capitalized and the other not then? I.e. do you want me to only change it to `Navigate`? Could also add a duplicate for `views` called `Views`, and deprecate the lowercase in a later version...